contents/docs/session-replay/troubleshooting.mdx

```diff
@@ -54,14 +54,6 @@ Some ad/tracking blockers will block PostHog from fetching `posthog-js`. If you'
 
 If you're having issues with recordings not looking correct, there are a few things you can:
 
-### Message too large warning
```

Comment:
what is the adoption of the SDK versions after implementing the compressing bits? older SDK versions might still face this issue, wondering if keeping it around until we don't see such reports for a couple of months?
just food for thought, not a blocker.

Reply:
rust capture doesn't support the backend code needed to make this work so we only had it for a few weeks...

but volume of message too large in capture is down by orders of magnitude so I think we're ok here 🤞